plugins/inputs/synproxy/synproxy_notlinux.go

```diff
@@ -0,0 +1,3 @@
+// +build !linux
+
+package synproxy
```

Comment:
For unsupported platforms create an empty plugin that prints an error, the ethtool pr has a really nice example:

https://github.com/influxdata/telegraf/pull/5865/files#diff-980231a87c3954198c19881c18fd0126

Reply:
Thanks, using an empty plugin which prints an error.